tests/appsec/rasp/test_shi.py

```diff
@@ -173,10 +173,10 @@ def test_ssrf_telemetry(self):
 
 
 @rfc("https://docs.google.com/document/d/1vmMqpl8STDk7rJnd3YBsa6O9hCls_XHHdsodD61zr_4/edit#heading=h.mshauo3jp6wh")
```

Comment:
is this the correct RFC ?

Reply:
I don't think so, it should be the same that we have here: https://github.com/DataDog/system-tests/pull/3215/files#diff-605b24a0db1fc8609eaaf3f6521b33356179a5e4174a62bd5d071e7ec5dfa54eR82

```suggestion
@rfc("https://github.com/DataDog/system-tests/pull/3215/files#diff-605b24a0db1fc8609eaaf3f6521b33356179a5e4174a62bd5d071e7ec5dfa54eR82")
```

Reply:
Yes, it is `[RFC] Exploit prevention in the ASM libraries`, pointing to `Local & Remote Configuration` section where capabilities are described. What's the problem?

Reply:
Got it, didnt know there was an specific RFC for SHi